.github/workflows/assign_fork_prs.yml

```diff
@@ -6,7 +6,7 @@ on:
 
 jobs:
   assign:
-    runs-on: self-hosted-docker-tiny
+    runs-on: ubuntu-24.04
```

Comment:
can we move the final result stuff? That one seems quite slow often and there seems to be no reason why it has to be on our runners. I suspect the hosted ones probably work better there.